### ISSUE TITLE
Video export presets - Video Name

### ISSUE THREAD

**Issue by pathumego** [mooniak/animate-svg-process#6]
Copied from `proposal.md`
- [ ] Video Name
-  [x] Video destination 
- [x] Service Presets
  - Vimeo [spec](https://vimeo.com/help/compression)
           - Size
           - HD / SD
  - YouTube  [spec](https://support.google.com/youtube/answer/1722171?hl=en)
           - Size
           - HD / SD
  - Facebook  [spec](https://www.facebook.com/help/124738474272230)
           - Size
           - HD / SD
- [x] Keep images or Remove
  - Warn about the extra hd storage

**Comment by pathumego** [comment]
@lpsandaruwan What do you think? Ive listed other options that can be implemented as extra in #7 and #8

**Comment by lpsandaruwan** [comment]
Using presets is a nice move. Will be more user friendly.

**Comment by pathumego** [comment]
Yep SO lets first implement the presets. Custom options can come later.

**Comment by lpsandaruwan** [comment]
okay. I'll keep the repo updated.
